src/main/java/com/cburch/logisim/fpga/hdlgenerator/AbstractHDLGeneratorFactory.java

```diff
@@ -528,6 +524,18 @@ public String getComponentStringIdentifier() {
     return Contents.get();
   }
 
+  private String getInstanceIdentifier(Object componentInfo, Long componentId) {
+    if (componentInfo != null && componentInfo instanceof NetlistComponent) {
```

Comment:
`componentInfo != null` check is not needed as `instanceof` is an operator and is safe to hand it a `null`. [Docs read](https://docs.oracle.com/javase/specs/jls/se11/html/jls-15.html#jls-15.20.2):

> At run time, the result of the instanceof operator is true if the value of the RelationalExpression is not null and the reference could be cast to the ReferenceType without raising a ClassCastException.

Reply:
Fixed